<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dlgDetachBarcode</class>
 <widget class="QDialog" name="dlgDetachBarcode">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>410</width>
    <height>480</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>410</width>
    <height>480</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Detach Barcode</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_Detach_Barcode">
   <property name="leftMargin">
    <number>6</number>
   </property>
   <property name="topMargin">
    <number>6</number>
   </property>
   <property name="rightMargin">
    <number>6</number>
   </property>
   <property name="bottomMargin">
    <number>6</number>
   </property>
   <item>
    <widget class="themeToolGroupBox" name="groupBox_Detach_Barcode">
     <property name="title">
      <string>Detach Barcode</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_groupBox_Detach_Barcode">
      <item>
       <widget class="QListWidget" name="listWidget_Barcodes">
        <property name="defaultDropAction">
         <enum>Qt::IgnoreAction</enum>
        </property>
        <property name="alternatingRowColors">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_Barcode">
        <property name="maxLength">
         <number>13</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="themePushButton" name="pushButton_Apply">
        <property name="text">
         <string>Apply</string>
        </property>
        <property name="icon">
         <iconset resource="resources.qrc">
          <normaloff>:/icon/icons/confirm.svgz</normaloff>:/icon/icons/confirm.svgz</iconset>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>themePushButton</class>
   <extends>QPushButton</extends>
   <header>theme.hpp</header>
  </customwidget>
  <customwidget>
   <class>themeToolGroupBox</class>
   <extends>QGroupBox</extends>
   <header>theme.hpp</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
